projects/vdk-plugins/vdk-meta-jobs/tests/test_meta_job.py

```diff
@@ -327,6 +330,48 @@ def test_meta_job_duplicate_jobs(self):
     def test_meta_job_not_allowed_job_key(self):
         self._test_meta_job_validation("meta-job-not-allowed-job-key")
 
+    def test_meta_job_wrong_job_arguments_type(self):
+        self._test_meta_job_validation("meta-job-wrong-job-arguments-type")
+
+    def test_meta_job_arguments(self):
+        self._set_up()
+        with mock.patch.dict(
+            os.environ,
+            self.env_vars,
+        ):
+            self.runner = CliEntryBasedTestRunner(plugin_entry)
+            result = self._run_meta_job("meta-job-arguments")
+            cli_assert_equal(0, result)
+            job2_post_req = [
+                req
+                for req, res in self.httpserver.log
+                if req.method == "POST"
+                and req.path.split("/jobs/")[1].split("/")[0] == "job2"
+            ]
+            job2_arguments = job2_post_req[0].json["args"]
+            assert len(job2_arguments) == 2
+            assert job2_arguments == {"table_name": "test_table", "counter": 123}
+            self.httpserver.stop()
+
+    def test_meta_job_empty_arguments(self):
+        self._set_up()
+        with mock.patch.dict(
+            os.environ,
+            self.env_vars,
+        ):
+            self.runner = CliEntryBasedTestRunner(plugin_entry)
+            result = self._run_meta_job("meta-job-empty-arguments")
+            cli_assert_equal(0, result)
+            job2_post_req = [
+                req
+                for req, res in self.httpserver.log
+                if req.method == "POST"
+                and req.path.split("/jobs/")[1].split("/")[0] == "job2"
+            ]
+            job2_arguments = job2_post_req[0].json["args"]
```

Comment:
extraxft as a helper function _get_job_arguments(httpserver) for example. 

The same code is used in two places.

Reply:
Done.